test/integration/commands/excel2xml/test_excel2xml_cli.py

```diff
@@ -7,85 +6,176 @@
-            self.assertTrue(len(catched_warnings) > 0)
-            messages = [str(w.message) for w in catched_warnings]
-            self.assertTrue(any(regex.search(_regex, msg) for msg in messages), msg=f"Failed with file '{file}'")
-
-        error_cases = [
-            (
-                f"{invalid_prefix}/id-propname-both.xlsx",
-                "Exactly 1 of the 2 columns 'id' and 'prop name' must be filled",
-            ),
-            (
-                f"{invalid_prefix}/id-propname-none.xlsx",
-                "Exactly 1 of the 2 columns 'id' and 'prop name' must be filled",
-            ),
-            (
-                f"{invalid_prefix}/nonexisting-proptype.xlsx",
-                "Invalid prop type",
-            ),
-            (
-                f"{invalid_prefix}/start-with-property-row.xlsx",
-                "The first row must define a resource, not a property",
-            ),
-        ]
-        for file, _regex in error_cases:
-            with self.assertRaisesRegex(BaseError, _regex, msg=f"Failed with file '{file}'"):
-                excel2xml_cli.excel2xml(file, "1234", "excel2xml-invalid")
+INVALID_EXCEL_DIRECTORY = "testdata/invalid-testdata/excel2xml"
+
+
+@pytest.fixture()
+def test_xml() -> str:
+    with open("testdata/excel2xml/excel2xml-expected-output.xml", encoding="utf-8") as f:
+        return f.read()
+
+
+class TestDifferentExtensions:
+    def test_xlsx(self, test_xml: str) -> None:
+        excel2xml_cli.excel2xml("testdata/excel2xml/excel2xml-testdata.xlsx", "1234", "excel2xml-output")
+        with open("excel2xml-output-data.xml", encoding="utf-8") as f:
+            returned = f.read()
+            assert returned == test_xml
+        Path("excel2xml-output-data.xml").unlink(missing_ok=True)
+
+    def test_xls(self, test_xml: str) -> None:
+        excel2xml_cli.excel2xml("testdata/excel2xml/excel2xml-testdata.xls", "1234", "excel2xml-output")
+        with open("excel2xml-output-data.xml", encoding="utf-8") as f:
+            returned = f.read()
+            assert returned == test_xml
+        Path("excel2xml-output-data.xml").unlink(missing_ok=True)
+
+    def test_csv(self, test_xml: str) -> None:
+        excel2xml_cli.excel2xml("testdata/excel2xml/excel2xml-testdata.csv", "1234", "excel2xml-output")
+        with open("excel2xml-output-data.xml", encoding="utf-8") as f:
+            returned = f.read()
+            assert returned == test_xml
+        Path("excel2xml-output-data.xml").unlink(missing_ok=True)
+
+
+class TestInvalidFiles:
+    def test_double_bool(self) -> None:
+        file = f"{INVALID_EXCEL_DIRECTORY}/boolean-prop-two-values.xlsx"
+        expected_msg = (
+            "A <boolean-prop> can only have a single value, but resource "
+            "'test_thing_1', property ':hasBoolean' (Excel row 6) contains more than one value."
+        )
+        _, catched_warnings = excel2xml_cli.excel2xml(file, "1234", "excel2xml-invalid")
```

Comment:
I realized that in the docstring of this function, only 1 return value is documented. This didn't happen in your PR, but could you fix it anyways?